docs/sections/cn_workflows.md

```diff
@@ -359,6 +359,65 @@ popper run -f sample.yml -c config.yml
 
 This runs the step `one` locally in the host and step `two` through slurm on 2 nodes.
 
+### Kubernetes
+
+Popper can be run on a cluster of multiple containers using Kubernetes as the fundamental resource manager to orchestrate the execution of a step to several nodes.
```

Comment:
- "cluster of multiple containers" is a bit confusing. maybe change to "Kubernetes cluster" and then add a hyperlink to kubernetes.

- we can remove "fundamental"

- a step runs on a single node, so "execution of a step to several nodes" is incorrect